script/pop.js

```diff
@@ -0,0 +1,124 @@
+    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    seeLivelink: '#',
+    seeSourceLink: '#',
+  },
+];
+
+const popButtons = document.getElementsByClassName('view');
+const work = document.querySelector('.cards-radious');
+const popup = document.createElement('section');
+popup.classList.add('popup-body');
+const overlay = document.createElement('div');
+overlay.setAttribute('id', 'overlay');
+const { body } = document;
+
+function popupDisplay(cardNum) {
+  body.style.overflowY = ('hidden');
+  overlay.style.display = ('block');
+  popup.style.display = ('block');
+  work.appendChild(overlay);
+  work.appendChild(popup);
+  popup.innerHTML = `
+        <div class="tonic-popup">
+          <h2>${projects[cardNum].title}</h2>
+          <i id="closeBtn" class="fas fa-times" alt='#'></i>
+        </div>
+        <ul class="list-popup">
+          <h2 class="canopy">CANOPY</h2>
+          <li class="date"> <i class="fa fa-circle"></i>&#160; Back End Dev</li>
+          <li class="date"> <i class="fa fa-circle"></i>&#160; 2015</li>
+        </ul>
+        <div class="card-img">
+          <img src="${projects[cardNum].src}" alt="Project-preview-1">
+        </div>
+        <div class="media-img">
+        <img src="${projects[cardNum].src}" alt="Project-preview-1">
+      </div>
+      <div class='flex'>
+          <p class="content-2">${projects[1].description}</p>
+          <p class="popup-content">${projects[0].description}</p>
+          <div class="flex-buttons">
+            <ul class="popup-btns">
+              <li>
+              <p>${projects[cardNum].tags[0]}</p>
+              </li>
+              <li>
+              <p>${projects[cardNum].tags[1]}</p>
+              </li>
+              <li>
+              <p>${projects[cardNum].tags[2]}</p>
+              </li>
+            </ul>
+
+            <ul class="popup-btns-media">
+            <li>
+            <p>${projects[cardNum].tags[0]}</p>
+            </li>
+            <li>
+            <p>${projects[cardNum].tags[1]}</p>
+            </li>
+            <li>
+            <p>${projects[cardNum].tags[2]}</p>
+            </li>
+            <li>
+            <p>${projects[cardNum].tags[3]}</p>
+            </li>
+            <li>
+            <p>${projects[cardNum].tags[4]}</p>
+            </li>
+            <li>
+            <p>${projects[cardNum].tags[5]}</p>
+            </li>
+          </ul>
+            <div class='popup-buttons'>
+              <a href="${projects[cardNum].seeLivelink}" class="view-popup"> <p> See live&#160; <img src="image/Arrow-Export.svg" alt="arrow icon">
+              </p> </a>
+              <a href="${projects[cardNum].seeSourceLink}" class="view-popup"> <p> See Source&#160; <i class="fab fa-github"></i></p> </a>
```

Comment:
- [ ] OPTIONAL : I recommend you to decrease the font size of these buttons on mobile screens, otherwise the content is wrapping: 
![image](https://user-images.githubusercontent.com/77038610/152639025-53b8bc18-7368-4030-8e0e-20a424ba60dc.png)
- [ ] OPTIONAL: I suggest you to change the color of GitHub icon to make the color of both icons the same, currently they are different: 
![image](https://user-images.githubusercontent.com/77038610/152639055-0369632c-cf68-4b4e-bf9b-025bdf5c1323.png)

- [ ] OPTIONAL : I suggest you to add a border between the icons and languages as provided in Figma:

Figma  | Yours
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/152639249-7f227516-3f6b-49c4-996e-e29098d164b1.png)| ![image](https://user-images.githubusercontent.com/77038610/152639246-02af3f17-df4f-4865-ab24-613f001aef45.png)